test/node-rpc-test.js

```diff
@@ -68,11 +69,29 @@ describe('RPC', function() {
     assert.deepEqual(info.localservicenames, ['NETWORK', 'WITNESS']);
   });
 
+  it('should rpc getpeerinfo', async () => {
+    const info = await nclient.execute('getpeerinfo', []);
+    assert.deepEqual(info, []);
+  });
+
+  it('should rpc getconnectioncount', async () => {
+    const connectionsCnt = await nclient.execute('getconnectioncount', []);
+    assert.strictEqual(connectionsCnt, 0);
+  });
```

Comment:
For these tests, it'd be more valuable if there were additional `it` blocks that actually connected another peer, tested again, disconnected, then tested again.

What I'd recommend here is adding a new `describe` block at line 50 (just after the `after` block) so this will be a subsection of tests right at the top.

```js
describe('Networking', function() {
  // Create a second full node with different ports
  const peer = new FullNode({
    memory: true,
    workers: false,
    port: 10000,
    httpPort: 20000,
    only: ['127.0.0.1']
  });

  // your existing tests go here (confirm 0 peers)

  it('should connect to peer', async () => {
    await peer.open();
    await peer.connect();
  });

  // now do your two tests again, but this time there should be more interesting output

  it('should disconnect peer', async () => {
    await peer.close();
  });

  // do the tests one last time and ensure we are back to 0
}

Reply:
I tried this, but both nodes are not connecting with each other !!
`getconnectioncount` still showing zero peers.

Reply:
So I pushed a commit to this branch: bfadb8d4d32d815c6a9e2e23ac64f36d5c9897d7

(By default, pull requests give repo maintainers permission to make changes, even though the branch is "yours")

I promise not to do this too often ;-) but I actually had to experiment a bit with this as well, take a look at the commit I added and you'll see a few things I forgot to add in my suggestion above:

`listen: true` -- by default a bcoin `FullNode` object does not listen for inbound connections. 
`await node.connect()` -- although the primary `FullNode` in this test is `open()` and operating, it's networking module is not activated.

You can see both of these properties being set in https://github.com/bcoin-org/bcoin/blob/master/bin/node which is the script that is run when you install bcoin and just run it from the command line with the command `bcoin`. In these tests, we are instantiating a `FullNode` object, but don't change many of the default options unless it is necessary.